runtime/testy/Ansible.py

```diff
@@ -104,10 +129,13 @@ def package(state):
                 self.sendBuffer.replace(packState)
                 nextCall += 1.0/self.packagerHZ
                 if (nextCall > time.time()):
-                    time.sleep(nextCall - time.time())
-            except Exception:
+                    try:
+                        time.sleep(nextCall - time.time())
+                    except ValueError:
```

Comment:
Why the check for `ValueError`? I'm guessing it's because the value can be negative. In that case, the `if (nextCall > time.time())` check above is redundant, and you can remove it and replace with a comment explaining that the try-catch handles that case.